files/ko/web/javascript/reference/errors/illegal_character/index.md

```diff
@@ -68,14 +68,14 @@ var colors = ['#000', '#333', '#666'];
 다른 곳에서 코드를 복사하여 붙여넣은 경우 잘못된 문자가 포함돼 있을 수 있습니다. 주의하세요!
 
 ```js example-bad
-var foo = 'bar';​
+var foo = 'bar';
```

Comment:
This line should actually be left alone; it's an example regarding the zero-width space character (0x200B) itself!

Reply:
oh, I should have been more careful. thanks!

Reply:
All good; I almost missed it in the Spanish locale as well when doing a search-and-replace, haha!